cli/config/configuration-qemu-virtiofs.toml.in

```diff
@@ -243,6 +243,8 @@ valid_file_mem_backends = @DEFVALIDFILEMEMBACKENDS@
 # This option changes the default hypervisor and kernel parameters
 # to enable debug output where available. This extra output is added
 # to the proxy logs, but only when proxy debug is also enabled.
+# Note: Debug output will only be put into qemu.log in the event
+# of a virtual hardware issue.
```

Comment:
`, otherwise it will be empty` ?  - I'd add this just to be clear

Reply:
added, thanks!